src/util/help_formatter.h

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************\
+
+Module: Help Formatter
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+/// \file
+/// Help Formatter
+
+#ifndef CPROVER_CPROVER_HELP_FORMATTER_H
+#define CPROVER_CPROVER_HELP_FORMATTER_H
```

Comment:
Now that this is in util: should be `CPROVER_UTIL_HELP_FORMATTER_H`

Reply:
done